### PR TITLE
Review Caesar Cipher 

### PR DESCRIPTION
**Pull request by becca6223**
thanks for reviewing!

### CODE REVIEW

File: caesar.c [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+int _CheckChar(char chr);
+int _CheckNumOfShifts(int num, int* ALPshift, int* DGshift);
+void _EncodeChar(char* add_char, int shiftDirect, int modValue, int numShift, int arraySize);
+
+//Function Definitions 
+Input GetInputStringInfo(char* string)
+{
+	Input stringInfo;
+	//Get number of shifts
+	stringInfo.numShift = atoi(strtok(string, ":"));
```
Comment (dsarpang) [inline caesar.c]: what is string is null? Also, avoid choosing a variable name that is commonly a reserved word in most languages

File: caesar.c [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+	
+	//Get the actual Input string and malloc it 
+	char* actstring = strtok(NULL, "\0");
```
Comment (dsarpang) [inline caesar.c]: Prefer verbose variable names or use common abbreviations. I.e. "actualString" is more readable

File: caesar.c [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+	{
+		return(LWALP);
+	}
+	else
+	{
+		return(NEITHER);
+	}
+}
+
+void _EncodeChar(char* add_char, int shiftDirect, int modValue, int numShift, int arraySize)
```
Comment (dsarpang) [inline caesar.c:70]: Nice job creating a helper function for this!

**Review by dsarpang — CHANGES_REQUESTED** [review]
We'll cover some of the review in class. My main feedback I see is that some of the variable names are a bit confusing (remember the 'readable' aspect of good code we learned in week1). Try to think more about what meaningful names are and I'll review another iteration. Nice work though!

File: caesar.c (hunk 1 above)
Comment (becca6223) [reply caesar.c]: Do you mean what if the string is NULL?

**Pull request merged**